src/main/java/mate/academy/webshop/service/impl/ProductServiceImpl.java

```diff
@@ -0,0 +1,42 @@
+package mate.academy.webshop.service.impl;
+
+import java.util.List;
+import java.util.NoSuchElementException;
+import mate.academy.webshop.dao.ProductDao;
+import mate.academy.webshop.lib.Inject;
+import mate.academy.webshop.lib.Service;
+import mate.academy.webshop.model.Product;
+import mate.academy.webshop.service.ProductService;
+
+@Service
+public class ProductServiceImpl implements ProductService {
+    @Inject
+    private ProductDao productDao;
+
+    @Override
+    public Product create(Product product) {
+        return productDao.create(product);
+    }
+
+    @Override
+    public Product get(Long id) {
+        return productDao.get(id)
+                .orElseThrow(NoSuchElementException::new);
```

Comment:
```suggestion
        return productDao.get(id);
```
давай поки залишимо так, ми потім це будемо рефакторити

Reply:
Ok!